front/public/locales/fr/simulation.json

```
@@ -79,5 +79,10 @@
     "tailPosition": "Queue"
   },
   "trainList": "Liste des trains",
-  "waiting": "Chargement en cours…"
+  "waiting": "Chargement en cours…",
+  "waypointsModal": {
+    "name": "nom",
+    "ch": "CH",
```

Suggestion, maybe we should start to use everywhere secondaryCode instead of CH
```suggestion
    "secondaryCode": "CH",
```

---

Good idea, done.